src/dataconnect/types.ts

```diff
@@ -135,7 +135,7 @@ export interface Generate {
 export interface JavascriptSDK {
   outputDir: string;
   package: string;
-  packageJSONDir?: string;
+  packageJsonDir?: string;
```

Comment:
Separate change - this was just incorrect before

Reply:
SGTM